apps/omg_api/test/crypto_test.exs

```diff
@@ -89,4 +89,42 @@ defmodule OMG.API.CryptoTest do
                <<65, 86, 211, 52, 45, 92, 56, 90, 135, 210, 100, 249, 6, 83, 115, 53, 146, 0, 5, 129>>
              )
   end
+
+  test "encode16 funciton encodes only specified fields" do
```

Comment:
Missing egde cases (encode & decode):
* called with empty map-list
* fields list is empty
* what if specified fields doesn't exists in map